code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -947,7 +951,11 @@
 
 /mob/living/silicon/robot/updatehealth()
 	..()
-	if(health < maxHealth*0.5) //Gradual break down of modules as more damage is sustained
+	var/missingHealth = 200 - (health + 100)
+	///Ranges from moving 
+	var/speedPenalty = min(missingHealth / (maxHealth), 1.5)
```

Comment:
Won't this always be 1.5? Assuming missinghealth <= maxhealth, then missinghealth/maxhealth will always be <= 1

Reply:
No, because you're incorrect in thinking missinghealth will always be less than maxhealth. 
maxHealth is 100. missingHealth goes up to 200.

50/100 = 0.5
100/100 = 1.0
150/100 = 1.5
190/100 = 1.5 because min() picks 1.5 instead of 1.9.